docker/docker.go

```diff
@@ -77,6 +79,35 @@ func Run(image string, args ...Arg) error {
 
 }
 
+func GetContainer(containerName string) (*types.Container, error) {
+	ctx := context.Background()
+
+	cfg := &config{}
+
+	containerName = containerNameFromImage(containerName)
+
+	cli, err := newCLI(cfg, ui)
+	if err != nil {
+		return nil, err
+	}
+
+	return cli.getContainer(ctx, containerName)
+}
+
+func CreateNetwork(name string) (string, error) {
+	ctx := context.Background()
+
+	cfg := &config{}
+
+	cli, err := newCLI(cfg, ui)
+	if err != nil {
+		return "", err
+	}
+
+	nwName := containerNameFromImage(name)
```

Comment:
Maybe rename this method to `sanitizeName`. It's a little confusing now because we create a network name with it here.